src/Symfony/Component/Translation/Tests/Catalogue/TargetOperationTest.php

```diff
@@ -71,13 +71,45 @@ public function testGetResultWithMixedDomains()
     {
         $this->assertEquals(
             new MessageCatalogue('en', [
-                'messages+intl-icu' => ['a' => 'old_a'],
+                'messages' => ['a' => 'old_a'],
```

Comment:
Since this is fixing the broken test by saying _the test was wrong_ I now **really** need someone with a deep understanding of the code to look into this :see_no_evil: 

But my reasoning for why this is actually the correct and intended behavior was:
_Existing_ messages should _always_ stay in their current domain. Moving new messages to the intl-icu domain is (at least in recent versions) handled by `AbstractOperation::moveMessagesToIntlDomainsIfPossible()`, see https://github.com/symfony/symfony/blob/1629b5943ecbec5058e60ad4328e6717a2224cf5/src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php#L247 and https://github.com/symfony/symfony/blob/1629b5943ecbec5058e60ad4328e6717a2224cf5/src/Symfony/Component/Translation/Catalogue/AbstractOperation.php#L157-L183

This Test was introduced in b72b7d34132b8e954c4caec7accbc8706286aaeb which also should only have affected **new** messages. So was _in my interpretation_ already wrong at that point.